.github/workflows/devtools.yml

```diff
@@ -0,0 +1,63 @@
+name: DevTools
+
+on:
+  push:
+    branches: [master]
+  pull_request: # run on all PRs, not just PRs to a particular branch
+
+jobs:
+  smoke-devtools:
+    runs-on: macos-latest
+
+    steps:
+    - name: Set $DEPOT_TOOLS_PATH
+      run: echo ::set-env name=DEPOT_TOOLS_PATH::$GITHUB_WORKSPACE/depot-tools
+    - name: Set $DEVTOOLS_PATH
+      run: echo ::set-env name=DEVTOOLS_PATH::$GITHUB_WORKSPACE/devtools-frontend
+    - name: Set $BLINK_TOOLS_PATH
+      run: echo ::set-env name=BLINK_TOOLS_PATH::$GITHUB_WORKSPACE/blink_tools
+    - name: Set $PATH
+      run: echo ::set-env name=PATH::$DEPOT_TOOLS_PATH:$PATH
+
+    - name: git clone
+      uses: actions/checkout@v2
+      with:
+        path: lighthouse
+    
+    - name: Cache depot tools, blink tools and content shell
+      uses: actions/cache@v2
+      with:
+        path: |
+          ${{ env.DEPOT_TOOLS_PATH }}
+          ${{ env.BLINK_TOOLS_PATH }}
+          ${{ github.workspace }}/lighthouse/.tmp/chromium-web-tests/content-shells
+        key: ${{ runner.os }}-${{ hashFiles('lighthouse/lighthouse-core/scripts/chromium-web-tests/download-*') }}
```

Comment:
do you anticipate manually controlling the chromium hash to compare against as we update for the beginning?

Reply:
For DevTools: the specific revision was temporary because I just happened to discover a bug building ToT devtools yesterday. It's fixed now. Using ToT.

For Blink tools/catapult: I think we can move to ToT.

Either way, if we use the cache action, it will hold onto whatever revisions it got for a week before updating. I think of this as a feature ... it means that a temporary failure to build DevTools, like seen yesterday, won't fail us. A handful of other people reported the issue w. devtools too, so if we were pinned to a specific revision b/c of the cache action, it's likely it would have been resolved before we noticed.